tests/utils/couchdb-utils.js

```diff
@@ -59,17 +59,37 @@ export const contacts = () => docs.filter(doc => contactTypes.includes(doc.type)
 
 const getDbByDoc = (id) => Object.keys(docsByDb).filter(dnName => docsByDb[dnName].has(id));
 
-export const editDoc = async (doc) => {
+export const insertDocs = async (documents) => {
+  docs.push(...documents);
+  const db = getDb(dbNames[0]);
+  await db.bulkDocs(documents);
+};
+export const editDoc = async (doc, useOldRev = false) => {
   const dbName = getDbByDoc(doc._id);
   const db = getDb(dbName);
-  const existentDoc = await db.get(doc._id);
-  await db.put({ ...doc, _rev: existentDoc._rev });
+  let existentDoc;
+  if (useOldRev) {
+    existentDoc = await db.get(doc._id);
+    doc._rev = existentDoc._rev;
+  } else {
+    existentDoc = await db.get(doc._id);
+    doc._rev = existentDoc._rev;
```

Comment:
What's the difference between the two? it looks like both blocks are doing the same thing.